src/RSEF/object_creator/create_downloadedObj.py

```diff
@@ -137,12 +139,14 @@ def doi_to_downloadedDic(doi,output_dir):
 
 def dois_to_downloadedDics(dois_list, output_dir):
     result = {}
+    output_path = output_dir + "/" + "downloaded_metadata.json"
```

Comment:
Better to put this and the one above (like 51) in a "constants" file. Otherwise it's difficult to maintain

Reply:
yes